### PR TITLE
Dedicated server-related mappings

### PR DESCRIPTION
**Pull request by MisterPeModder**

### CODE REVIEW

File: mappings/net/minecraft/server/dedicated/AbstractPropertiesHandler.mapping [hunk 1]
```diff
@@ -0,0 +1,31 @@
+CLASS tv net/minecraft/server/dedicated/AbstractPropertiesHandler
```
Comment (liach) [inline mappings/net/minecraft/server/dedicated/AbstractPropertiesHandler.mapping:1]: Is this for parsing server.properties file?

Comment (MisterPeModder) [reply mappings/net/minecraft/server/dedicated/AbstractPropertiesHandler.mapping:1]: yes

File: mappings/net/minecraft/server/dedicated/ServerPropertiesHandler.mapping [hunk 2]
```diff
@@ -44,3 +44,6 @@ CLASS ts net/minecraft/server/dedicated/SeverProperties
 	FIELD x enableRcon Z
 	FIELD y rconPort I
 	FIELD z rconPassword Ljava/lang/String;
+	METHOD a load (Ljava/nio/file/Path;)Lts;
+	METHOD a createHandler (Ljava/util/Properties;)Lts;
```
Comment (asiekierka) [inline mappings/net/minecraft/server/dedicated/ServerPropertiesHandler.mapping]: Why not just create()?

**Review by asiekierka — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/server/dedicated/ServerPropertiesHandler.mapping (hunk 2 above)
Comment (MisterPeModder) [reply mappings/net/minecraft/server/dedicated/ServerPropertiesHandler.mapping]: Changed it

File: mappings/net/minecraft/server/dedicated/MinecraftDedicatedServer.mapping [hunk 3]
```diff
@@ -1,7 +1,12 @@
 CLASS tr net/minecraft/server/dedicated/MinecraftDedicatedServer
 	FIELD g LOGGER_DEDICATED Lorg/apache/logging/log4j/Logger;
+	FIELD h sha1Pattern Ljava/util/regex/Pattern;
```
Comment (asiekierka) [inline mappings/net/minecraft/server/dedicated/MinecraftDedicatedServer.mapping]: If this is static final (I'm not sure if it is), it should be WRITTEN_THIS_WAY

**Review by asiekierka — APPROVED** [review]

File: mappings/net/minecraft/server/dedicated/MinecraftDedicatedServer.mapping (hunk 3 above)
Comment (Juuxel) [reply mappings/net/minecraft/server/dedicated/MinecraftDedicatedServer.mapping]: It is a static final field.

Comment (MisterPeModder) [reply mappings/net/minecraft/server/dedicated/MinecraftDedicatedServer.mapping]: Oups, I didn't notice!

**Review by Gegy — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]
Looks good to me

👍

**Pull request merged**